backend/controllers/postController.js

```diff
@@ -0,0 +1,92 @@
+import mongoose from "mongoose"
+const ObjectId = mongoose.Types.ObjectId
+import Post from "../models/postModel.js"
+import Comment from "../models/commentModel.js"
+
+async function newPost(req, res, next) {
+    const {
+        content, 
+        takenWith, 
+        createdAt
```

Comment:
This is pre implemented in mongo since we are using the timestamp feild in the schema which by default will create and manage the createdAt and updatedAt 
![image](https://github.com/aldab2/family_space/assets/34516271/7d7406bc-4b0a-405d-a0d2-570e51730e12)
![image](https://github.com/aldab2/family_space/assets/34516271/048bdf73-9c7f-4fd5-ad46-5b494e45c7e4)